tools/rosbash/rosfish

```diff
@@ -1039,3 +1052,4 @@ complete -x -c rossrv         -a '(_roscomplete_rossrv)'
 complete -x -c roscreate-pkg  -a '(_roscomplete_roscreate_pkg)'
 complete -x -c roswtf         -a '(_roscomplete_roswtf)'
 complete -x -c rosconsole     -a '(_roscomplete_rosconsole)'
+complete -x -c rosmv          -a '(_roscomplete_file)'
```

Comment:
Please add a newline at the end of the file.

Reply:
Added new line